### ISSUE TITLE
Implementation Issues

### ISSUE THREAD

**Issue by althe3rd** [azmenak/react-stripe-checkout#58]
I am really loving the checkout component but I am finding myself completely stuck at getting the AJAX submission to work.  My submissions using ajax (using the syntax you use in your example) result in a 402 error on stripes end claiming no card data was sent.

Keep in mind that the ajax request is being posted to a php file that is already tested at handling charges normally when the checkout form is just submitted sans ajax.  The php used is exactly the same php stripe provides in their samples in their documentation. 

`<?php
  require_once('./config.php');

  $token  = $_POST['stripeToken'];

  $customer = \Stripe\Customer::create(array(
      'email' => 'customer@example.com',
      'source'  => $token
  ));

  $charge = \Stripe\Charge::create(array(
      'customer' => $customer->id,
      'amount'   => 5000,
      'currency' => 'usd'
  ));

?>`

So basically I am wondering if you have had any experience implementing this with php as the processing for charges and what did you have to do to implement it with ajax.  I seem to be a bit out of my depth as to figuring out whats wrong.

**Comment by althe3rd** [comment]
It took a lot of troubleshooting but I eventually figured out that the fetch method wasn't sending the correct post data, leaving everything else the same and using jQuery to send it completely fixed everything.
